packages/editor/src/lib/editor/types/event-types.ts

```diff
@@ -58,7 +58,6 @@ export type TLPointerEventInfo = TLBaseEventInfo & {
 	pointerId: number
 	button: number
 	isPen: boolean
-	source: 'user' | 'followingCamera'
```

Comment:
Since this is a one-off situation, I think that rather than changing our event interface, let's use `pointerId` as a way to sneakily share that this is an "artificial" pointer move.